nemo/utils/exp_manager.py

```diff
@@ -101,6 +101,48 @@ class ExpManagerConfig:
     checkpoint_callback_params: Optional[CallbackParams] = CallbackParams()
     # Additional exp_manager arguments
     files_to_copy: Optional[List[str]] = None
+    # logs timing of train/val/test steps
+    log_step_timing: Optional[bool] = True
+    # if True torch.cuda.synchronize() is called on start/stop
+    step_timing_sync_cuda: Optional[bool] = False
+
+
+class TimingCallback(Callback):
+    """
+    Logs execution time of train/val/test steps
+    """
+
+    def __init__(self, timer=None):
+        # support external timer
+        if timer is None:
+            timer = timers.NamedTimer(sync_cuda=step_timing_sync_cuda)
```

Comment:
No argument called step_timimg_sync_cuda passed to this init

Reply:
Fixed.